src/compiler/docs/readme/output-docs.ts

```diff
@@ -31,7 +31,9 @@ export const generateReadme = async (config: d.Config, compilerCtx: d.CompilerCt
   }));
 };
 
-export const generateMarkdown = (config: d.Config, userContent: string, cmp: d.JsonDocsComponent, cmps: d.JsonDocsComponent[], footer: string) => {
+export const generateMarkdown = (config: d.Config, userContent: string, cmp: d.JsonDocsComponent, cmps: d.JsonDocsComponent[], readmeOutput: d.OutputTargetDocsReadme) => {
+  //If the readmeOutput.dependencies is true or undefined the dependencies will be generated.
+  const dependencies = readmeOutput.dependencies === true || readmeOutput.dependencies === undefined ? depsToMarkdown(config, cmp, cmps) : [];
```

Comment:
```suggestion
  const dependencies = readmeOutput.dependencies !== false ? depsToMarkdown(config, cmp, cmps) : [];
```

Reply:
Will this not break backwards compatibility? If readmeOutput.dependencies is undefined then undefined !== false will evaluate to true

Reply:
After looking at it again I see that I got it backwards and that your suggestion is correct.